scss/_forms.scss

```diff
@@ -92,24 +92,23 @@
 //
 // Note that as of 8.3, iOS doesn't support `week`.
 
-@media screen and (-webkit-min-device-pixel-ratio: 0) {
-  input[type="date"],
-  input[type="time"],
-  input[type="datetime-local"],
-  input[type="month"] {
-    &.form-control {
-      line-height: $input-height;
-    }
+_::-webkit-full-page-media, // Hack to make this CSS be Safari-only; see http://browserbu.gs/css-hacks/webkit-full-page-media/
```

Comment:
Tweaked the SCSS-Lint config to squelch this. It was trying to enforce the dash-separated-words naming convention on *element names* in addition to class names.